Dockerfile

```diff
@@ -6,5 +6,5 @@ WORKDIR /app
 COPY package.json /app
 RUN npm install 
 COPY . /app 
-EXPOSE 3000
+EXPOSE 5000
 CMD ["npm", "run", "dev"]
```

Comment:
This script uses `nodemon`. I'd just use the `start` script, as it's likely you won't use this docker to develop code on it.